CHANGELOG.md

```diff
@@ -3,6 +3,81 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## [4.0.0-alpha.25](https://github.com/mui-org/material-ui-x/compare/v4.0.0-alpha.24...v4.0.0-alpha.25)
+
+_Apr 14, 2021_
+
+Big thanks to the 5 contributors who made this release possible. Here are some highlights ✨:
+
+- 🎁 Add boolean column type @m4theushw
+- ⚡️ Update to React 17 (#1331) @m4theushw
+- ♿ Make column header cells focusable (#1289), and fix roving tabindex (#1327) @DanailH
+- 🐛 Ignore event from portal in cells (#1324) @oliviertassinari
+- 🐞 Bugfixes
+
+### @material-ui/x-grid@v4.0.0-alpha.25 / @material-ui/data-grid@v4.0.0-alpha.25
+
+#### Breaking changes
+
+- [DataGrid] Add support for custom row ids without cloning (#1377) @m4theushw
```

Comment:
These two PRs are part of the same fix.

```suggestion
- [DataGrid] Add support for custom row ids without cloning (#1377, #1371) @m4theushw
```

Reply:
it's fine as they were merged as separate PR.  
We list the PR commits here, not the issues